Error installing remix-auth v2.1.1

I am trying to install remix-auth into a remix app and am receiving the following error:

```
➜  my-remix-app npm i remix-auth
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: remix-app-template@undefined
npm ERR! Found: remix@0.20.1
npm ERR! node_modules/remix
npm ERR!   remix@"^0.20.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer remix@"^0.19.2" from remix-auth@2.1.1
npm ERR! node_modules/remix-auth
npm ERR!   remix-auth@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/package.json
+++ b/package.json
@@ -44,16 +44,16 @@
   "peerDependencies": {
     "@remix-run/node": "^0.20.1",
     "@remix-run/react": "^0.20.1",
-    "remix": "^0.19.2"
+    "remix": "^0.20.1"
   },
   "devDependencies": {
     "@babel/core": "^7.14.2",
     "@babel/preset-env": "^7.14.1",
     "@babel/preset-react": "^7.13.13",
     "@babel/preset-typescript": "^7.13.0",
-    "@remix-run/dev": "^0.19.2",
-    "@remix-run/node": "^0.19.2",
-    "@remix-run/react": "^0.19.2",
+    "@remix-run/dev": "^0.20.1",
+    "@remix-run/node": "^0.20.1",
+    "@remix-run/react": "^0.20.1",
     "@types/jest": "^26.0.23",
     "@types/prop-types": "^15.7.4",
     "@types/react": "^17.0.14",
@@ -75,7 +75,7 @@
     "jest-fetch-mock": "^3.0.3",
     "prettier": "^2.3.2",
     "react": "^17.0.2",
-    "react-router-dom": "^6.0.0-beta.6",
+    "react-router-dom": "^6.0.0-beta.8",
     "remix": "^0.20.1",
     "ts-node": "^9.1.1",
     "typescript": "^4.3.5"

